backend/tracim_backend/applications/agenda/lib.py

```diff
@@ -696,7 +696,7 @@ def sync_user_agenda(self, user: User, context: TracimContext, create_event: boo
                 logger.exception(self, exc)
 
     def sync_workspace_symlinks(
-        self, role: UserRoleInWorkspace, context: TracimContext, role_deletion=False
+        self, role: UserConfigInWorkspace, context: TracimContext, role_deletion=False
```

Comment:
did not rename param role, is it normal?

Reply:
it have been changed by user_workspace_config